lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -48,10 +48,10 @@ function E2Lib.setAng(ent, ang)
 end
 
 --Blacklist format:
---<top folder the material is in>[%./\\]+<material name>
+--<top folder the material is in>[%./\\]+<material name>$
 --Should prevent work-arounds like pp/./copy pp/./././copy pp\\copy etc.
 local material_blacklist = {
-	"pp[%./\\]+copy"
+	"pp[%./\\]+copy$"
 }
 local function validMaterial(material)
 	local lower = string.lower(material)
```

Comment:
Not to mention that lowercasing is a wrong thing to do, but this behavior is being inherited from the original code, so I don't blame you.
Are you wondering why it is wrong?
No problem, let me enlighten you:
Linux by nature is a case-sensitive system. Windows, on the other hand, is case-insensitive (by default), **but it can be made case-sensitive** as well.
Are you wondering why it matters?
It means for a case-sensitive system, that `pp/copy`, `Pp/copy`, `pP/copy` and `PP/copy` are **4** different materials!

But, you completely didn't see this coming, so I don't blame you. Anyways, whoever added the `string.lower` into this code is a *noob* (or it is just seriously old code when Linux didn't even have GMod yet, in which case I understand it).